.travis.yml

```diff
@@ -23,4 +31,4 @@ deploy:
   script: ./bin/deploy-travis
   skip_cleanup: true
   on:
-    branch: master
+    all_branches: true
```

Comment:
I assume this was changed to test the deployment process, but this should be reverted before this gets merged.